### ISSUE TITLE
Flags look to not be properly passed to compiler

### ISSUE THREAD

**Issue by piomis** [roozbehid/GCCTask#21]
I have following problem with additional include paths and additional defines
It looks that neither -I nor -D options are properly built for compiler call - when activated verbose log I can see just few spaces:

It does not work neither in my project nor in ConsoleApp from Examples:

```
                     Task Parameter:GCCToolSupportsResponsefile=True (TaskId:31)
                       ConsoleApplication.cpp (TaskId:31)
                       stdafx.cpp (TaskId:31)
                     g++ -fmax-errors=1000      -x c++ -g2 -gdwarf-2  -m32  -w -MM ConsoleApplication.cpp (TaskId:31)
                     g++ -fmax-errors=1000      -x c++ -g2 -gdwarf-2  -m32  -w -MM stdafx.cpp (TaskId:31)
                     g++ -fmax-errors=1000      -x c++ -g2 -gdwarf-2  -m32  -c ConsoleApplication.cpp -o "obj/x86/LinuxDebug/ConsoleApplication.o" (TaskId:31)
                     g++ -fmax-errors=1000      -x c++ -g2 -gdwarf-2  -m32  -c stdafx.cpp -o "obj/x86/LinuxDebug/stdafx.o" (TaskId:31)
```

On my system I have:
dotnet --version
6.0.200

and
gccbuildtargets: 1.19.325.5

Thanks in advance for any help.

**Comment by roozbehid** [comment]
Can you share your vcxproj file? Also have you tried adding them manually
over additional options.

**Comment by piomis** [comment]
Not sure what do you mean by "Additional Options".

Like I wrote I tried with my project. But I have the same behaviour with Example projects (Examples/VcxProj_Complex/ConsoleApplication) - in fact log from first post comes from build of that example.
AdditionalIncludeDirectories and PreprocessorDefinitions look to be ignored.

But just in case I attach also my project:
[libcunit.zip](https://github.com/roozbehid/dotnet-vcxproj/files/8216485/libcunit.zip)

**Comment by piomis** [comment]
OK - I think I have identified where problem is.

Looks that platform must not be provided as "Win32" - it has to be "x86" (or x64).
Even if dotnet build is called with -p:Platform=Win32 it is internally changed into x86.
Any settings in project with `Condition="'$(Configuration)|$(Platform)'=='LinuxDebug|Win32'"` won't work.
It has to be `Condition="'$(Configuration)|$(Platform)'=='LinuxDebug|x86'"`

By the way I also found few problems in llvm settings - are you interested? should I prepare pull-request?
